src/components/StoryPointsContainer.tsx

```diff
@@ -0,0 +1,79 @@
+'use client'
+
+import { socketRoomEmitter } from '@/services/socket'
+import { useSocketListener } from '@/services/socket'
+import { useState, useEffect } from 'react'
+import StoryPointButton from './StoryPointButton'
+import type { RoomInfoData } from './RoomMainUi'
+
+// export default function StoryPointsContainer({
+// 	setSelectedStoryPoint,
+// }: { setSelectedStoryPoint: (value: string) => void }) {
+export default function StoryPointsContainer({
+	roomId,
+	userName,
+}: RoomInfoData) {
+	const [selectedStoryPoint, setSelectedStoryPoint] = useState<string | null>(
+		null,
+	)
+	const [isPointBtnDisabled, setIsPointBtnDisabled] = useState(false)
+
+	// const resetStoryPointBtns = useSocketListener('reset-points')
+	const resetStoryPointBtns = useSocketListener('show-disable-reset-points')
+
+	useEffect(() => {
+		if (!selectedStoryPoint) return
+		const timeStamp = Date.now().toString()
+		socketRoomEmitter(
+			'story-points',
+			selectedStoryPoint,
+			userName,
+			timeStamp,
+			roomId,
+		)
+	}, [selectedStoryPoint, roomId, userName])
+
+	useEffect(() => {
+		if (!resetStoryPointBtns) return
+		console.log(
+			'%c>>> resetStoryPointBtns',
+			'color: red',
+			resetStoryPointBtns,
+		)
+		if (resetStoryPointBtns.message === 'true') {
+			setSelectedStoryPoint(null)
+			setIsPointBtnDisabled(true)
+		}
+		if (resetStoryPointBtns.message === 'false') {
+			setIsPointBtnDisabled(false)
+		}
+	}, [resetStoryPointBtns])
+
+	const storyPointValues = ['?', 0, 1, 2, 3, 5, 8, 13, 20, 40, 100]
```

Comment:
todo: This array is part of the game settings. In your future MR, this will need to be moved outside. Leave a code comment for you to not forget about it later on :)

```suggestion
  // TODO: Move this out to the game settings
	const storyPointValues = ['?', 0, 1, 2, 3, 5, 8, 13, 20, 40, 100]
```

Reply:
I know it needs to be in settings, but good suggestion to add a TODO. 
DONE